### DISCUSSION TITLE
Send pins to the note box in the dropdown

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#77] [Ideas]
Is it possible to send pins to the note box in the dropdown just like sending text to note as discussed [here](https://github.com/buggyj/tiddlyclip-plugin/discussions/49#discussioncomment-1970796).
ie, Pin2note instead of text2note

**Comment by buggyj** [comment]
this has not been implemented

**Reply by arunnbabu81** [reply]
> this has not been implemented

Is `pin2note ` a doable feature?

**Reply by buggyj** [reply]
pin copies the url of the picture, I don't see how coping multiple urls to the note box would be useful.

**Reply by arunnbabu81** [reply]
> pin copies the url of the picture, I don't see how coping multiple urls to the note box would be useful.

This is my usecase.
Sometimes I send text into the `note box` first and after doing some formatting or modifications only,  I will clip or send it into the wiki.
I have also find the need to send pins along with the text to `note box ` for the continuity of note taking from twitter or some articles. Otherwise I will have to send text from note box first and then only be able to send the pins to the wiki. So these pins which are added separately will line up in the bottom of the tiddler and natural continuity of the text and pins will be lost.

**Reply by buggyj** [reply]
It is possible. I will be working on the tiddlyclip addon in few month as I need to re-write part of it for the new chrome addon interface, and with look at it then.

**Reply by arunnbabu81** [reply]
> 



> It is possible. I will be working on the tiddlyclip addon in few month as I need to re-write part of it for the new chrome addon interface, and with look at it then.

I am badly missing this feature when I am taking notes from tweets. This feature would be of great help to increase the speed of note taking
